src/edit.js

```diff
@@ -0,0 +1,11 @@
+import { taskSection, todos } from './index.js'//eslint-disable-line
+
+export default function change() {
+  taskSection.addEventListener('change', (event) => {
+    if (event.target.className === 'checkBoxClass') {
+      todos[event.target.id].status = !todos[event.target.id].status;
+      localStorage.setItem('task', JSON.stringify(todos));
+      event.target.parentElement.classList.toggle('checked');
+    }
+  });
+}
```

Comment:
- [ ]  Good work up to this point, However, I tried to edit your task descriptions but I couldn't. Note that one should be able to edit a task description as a project requirement.

![image](https://user-images.githubusercontent.com/98453979/199973862-0f396da4-e310-49bb-8a57-a39c11d3398a.png)

- [ ] I noted that you had implemented the `clear all tasks` functionality :smile: Kindly note that that is supposed to be covered in the next milestone, `Todo list : interactive`. I would advise you to remove it for now so that your work will be reviewed properly in the next milestone.

Reply:
> frank1738

Hi, @frank1738 To edit you have to click on the item you want to edit then the item text will be displayed on the input tag for you to edit and update.